src/nbl/video/COpenGLCommandBuffer.cpp

```diff
@@ -115,8 +115,7 @@ COpenGLCommandBuffer::~COpenGLCommandBuffer()
         if (!IGPUCommandBuffer::canReset())
             return false;
 
-        freeSpaceInCmdPool();
-        m_commands.clear();
+        releaseResourcesBackToPool();
         return IGPUCommandBuffer::reset(_flags);
```

Comment:
if we rewrite to a `reset_impl` then the state transition can be common to all backends

Reply:
Resolved.